aspnetcore/blazor/components/cascading-values-and-parameters.md

```diff
@@ -742,7 +742,7 @@ Similar to a regular component parameter, components accepting a cascading param
 ```razor
 <main>
     <div class="top-row px-4">
-        <a href="https://docs.microsoft.com/aspnet/" target="_blank">About</a>
+        <a href="https://learn.microsoft.com/aspnet/" target="_blank">About</a>
```

Comment:
Revert this one, as it's locked into the 7.0 template.

https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/MainLayout.NoAuth.razor#L12

Reply:
Just reverting this one for the rebrand. Thanks for the feedback.